pkg/memory/config/config.go

```diff
@@ -16,5 +16,6 @@ package config
 
 // Configuration describes the options to customize the storage behavior
 type Configuration struct {
-	MaxTraces int `yaml:"max-traces" mapstructure:"max_traces"`
+	MaxTraces             int  `yaml:"max-traces" mapstructure:"max_traces"`
+	EnableStreamingWriter bool `yaml:"enable-streaming-writer" mapstructure:"enable_streaming_writer"`
```

Comment:
I am not sure this is a good place for this flag. Memory storage by itself has no need for this flag, it only makes sense in the context of the memstore-plugin.

Reply:
Got it, I deleted it here and kept it in the memory plugin